sklearn/cluster/tests/test_bicluster.py

```diff
@@ -218,19 +218,6 @@ def test_perfect_checkerboard(global_random_seed):
     assert consensus_score(model.biclusters_, (rows, cols)) == 1
 
 
-@pytest.mark.parametrize(
-    "params, type_err, err_msg",
-    [
-        ({"n_init": 0}, ValueError, "n_init == 0, must be >= 1."),
-        ({"n_init": 1.5}, TypeError, "n_init must be an instance of"),
-        (
-            {"n_clusters": "abc"},
-            TypeError,
-            "n_clusters must be an instance of",
-        ),
-        ({"svd_method": "unknown"}, ValueError, "Unknown SVD method: 'unknown'"),
-    ],
-)
 def test_spectralcoclustering_parameter_validation(params, type_err, err_msg):
```

Comment:
The error
```
E       fixture 'params' not found
```
is generated here by the fact that the parametrization has gone... 
Either you remove completely the test either you keep the parametrization at least for some parameters.

Reply:
Thanks, I have removed this test.